src/modules/85-cv/components/ExecutionVerification/components/DeploymentMetrics/DeploymentMetrics.tsx

```diff
@@ -399,7 +400,13 @@ export function DeploymentMetrics(props: DeploymentMetricsProps): JSX.Element {
           </Layout.Horizontal>
         </Layout.Horizontal>
       </Layout.Horizontal>
-      <Container className={css.content}>{renderContent()}</Container>
+      <Container
+        className={cx(css.content, {
+          [css.whiteBackground]: (!currentViewData?.length && !loading) || getShouldShowError(error, shouldUpdateView)
```

Comment:
NIT: We can use defaultTo 

```suggestion
          [css.whiteBackground]: defaultTo(!currentViewData?.length && !loading,  getShouldShowError(error, shouldUpdateView))
```

Reply:
No @deepesh-ui. `defaultTo` doesn't work like OR operator (||) And it only returns the default value if the first value is null, NaN, or undefined. Here we will get the first value as a boolean and if we get it as false, we want to check the second one.